netsim/modules/routing.py

```diff
@@ -868,7 +867,7 @@ def check_static_routes(idx: int,o_name: str,node: Box,topology: Box) -> None:
     for af in log.AF_LIST:
       if af not in sr_data:
         continue
-      for nh_entry in sr_data.nexthop.nhlist:
+      for nh_entry in sr_data.nexthop.nhlist[:sr_features.get('max_nexthop',256)]:
         sr_entry = { af: sr_data[af], 'nexthop': nh_entry }
```

Comment:
It may be easier to make ```nexthop``` a list - then the Jinja2 template can pick it up from there